doc/reference/backends.rst

```diff
@@ -2,18 +2,26 @@
 Backends and Configs
 ********************
 
-.. note:: Both NetworkX backend and config systems are experimental. They
-   let you execute an alternative backend implementation instead of NetworkX's
-   pure Python dictionaries implementation. Things will almost certainly
-   change and break in future releases!
+Backends let you execute an alternative backend implementation instead of NetworkX's
+pure Python dictionaries implementation. Configs provides library level storage
+of configuration settings that otherwise might come from environment variables.
+
+.. note:: Both NetworkX backend and config systems are new and and actively developed,
+   with frequent updates and improvements. While the backend user interface is unlikely
+   to make breaking changes, the backend developers may see some need to update
+   their code (within a deprecation period). The NetworkX backend developers should
+   regularly monitor the new updates to maintain compatibility. Participating in weekly
```

Comment:
How about this?
```suggestion
of configuration settings that can also come from environment variables.

.. note:: Both NetworkX backend and config systems are receiving frequent updates
   and improvements. The user interface for using backends is generally stable.
   In the unlikely case where breaking changing are necessary to the backend or
   config APIs, the standard deprecation policy of NetworkX may not be followed.
   This flexibility is intended to allow us to respond rapidly to user feedback
   and improve usability, and care will be taken to avoid unnecessary disruption.
   Developers of NetworkX backends should regularly monitor updates to maintain
   compatibility. Participating in weekly
   `NX-dispatch meetings <https://scientific-python.org/calendars/networkx.ics>`_
   is an excellent way to stay updated and to contribute to the ongoing discussions.
```
It might be nice to link to the deprecation policy page when mentioning the deprecation policy:
https://networkx.org/documentation/latest/developer/deprecations.html